src/outputs/outputs_main.py

```diff
@@ -219,4 +220,10 @@ def run_outputs(  # noqa: C901
         output_intram_totals(intram_tot_dict, config, write_csv)
         OutputMainLogger.info("Finished Intramural totals output.")
 
+    # Running National Accounts output
+    if config["global"]["output_pnp_na"]:
+        OutputMainLogger.info("Starting PNP National Accounts output...")
```

Comment:
if config["global"]["output_pnp_na"] and config["survey"]["survey_type"] == "PNP":
```suggestion
    if config["global"]["output_pnp_na"] :
```